specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2025-08-01-preview/examples/RedisEnterpriseMigrationGet.json

```diff
@@ -0,0 +1,21 @@
+{
+  "parameters": {
+    "clusterName": "cache1",
+    "resourceGroupName": "rg1",
+    "api-version": "2025-08-01-preview",
+    "subscriptionId": "e7b5a9d2-6b6a-4d2f-9143-20d9a10f5b8f"
+  },
+  "responses": {
+    "200": {
+      "body": {
```

Comment:
if migrations/default is a resource, then this payload needs to look like a resource (in terms of conventions) and include the required 'id' and 'type' properties, and put custom properties inside a 'properties' bag